The doc subpkg of openblas fails the doc test

The documentation subpackage of openblas fails testing with the following error:

```
2025/02/04 21:56:42 WARN + grep -q ^Cflags: usr/lib/pkgconfig/openblas.pc
2025/02/04 21:56:42 WARN + pkgconf --cflags openblas
2025/02/04 21:56:42 WARN + '[' true '=' false ]
2025/02/04 21:56:42 WARN + exit 0
2025/02/04 21:56:42 INFO running step "test/ldd-check"
2025/02/04 21:56:42 WARN + '[' -d /home/build ]
2025/02/04 21:56:42 WARN + cd /home/build
2025/02/04 21:56:42 WARN + exit 0
2025/02/04 21:56:42 INFO running step "check for missing library dependencies using ldd"
2025/02/04 21:56:42 INFO [ldd-check] Testing binaries in package openblas-dev
2025/02/04 21:56:42 WARN + '[' -d /home/build ]
2025/02/04 21:56:42 WARN + cd /home/build
2025/02/04 21:56:42 WARN + set +x
2025/02/04 21:56:42 INFO PASS[ldd-check]: /usr/lib/liblapack.so
2025/02/04 21:56:42 INFO PASS[ldd-check]: /usr/lib/liblapacke.so
2025/02/04 21:56:42 INFO PASS[ldd-check]: /usr/lib/libopenblas.so
2025/02/04 21:56:43 INFO INFO[ldd-check]: tested 3 files with ldd. 3 passes. 0 fails.
2025/02/04 21:56:43 INFO running test pipeline for subpackage openblas-doc
2025/02/04 21:56:43 INFO building test workspace in: '/home/user/tmp/melange-guest-2129895668-openblas-doc' with apko
2025/02/04 21:56:43 [DEBUG] GET https://packages.wolfi.dev/os/apk-configuration
2025/02/04 21:56:43 INFO setting apk repositories: [/home/user/src/wolfi-os/packages https://packages.wolfi.dev/os]
2025/02/04 21:56:43 INFO image configuration:
2025/02/04 21:56:43 INFO   contents:
2025/02/04 21:56:43 INFO     build repositories: []
2025/02/04 21:56:43 INFO     runtime repositories: []
2025/02/04 21:56:43 INFO     keyring:      []
2025/02/04 21:56:43 INFO     packages:     [openblas-doc apk-tools grep man-db texinfo]
2025/02/04 21:56:43 INFO installing ca-certificates-bundle (20241121-r1)
2025/02/04 21:56:43 INFO installing wolfi-baselayout (20230201-r16)
2025/02/04 21:56:43 INFO installing glibc (2.40-r8)
2025/02/04 21:56:43 INFO installing ld-linux (2.40-r8)
2025/02/04 21:56:43 INFO installing libgcc (14.2.0-r8)
2025/02/04 21:56:43 INFO installing glibc-locale-posix (2.40-r8)
2025/02/04 21:56:43 INFO installing zlib (1.3.1-r5)
2025/02/04 21:56:43 INFO installing libcrypto3 (3.4.0-r6)
2025/02/04 21:56:43 INFO installing libssl3 (3.4.0-r6)
2025/02/04 21:56:43 INFO installing apk-tools (2.14.4-r2)
2025/02/04 21:56:43 INFO installing libpcre2-8-0 (10.44-r2)
2025/02/04 21:56:43 INFO installing grep (3.11-r2)
2025/02/04 21:56:43 INFO installing libpipeline (1.5.8-r0)
2025/02/04 21:56:43 INFO installing libbz2-1 (1.0.8-r10)
2025/02/04 21:56:43 INFO installing libxcrypt (4.4.38-r0)
2025/02/04 21:56:43 INFO installing libcrypt1 (2.40-r8)
2025/02/04 21:56:43 INFO installing perl (5.40.1-r0)
2025/02/04 21:56:43 INFO installing libstdc++ (14.2.0-r8)
2025/02/04 21:56:43 INFO installing groff (1.23.0-r3)
2025/02/04 21:56:43 INFO installing libseccomp (2.6.0-r0)
2025/02/04 21:56:43 INFO installing gdbm (1.24-r2)
2025/02/04 21:56:43 INFO installing man-db (2.13.0-r1)
2025/02/04 21:56:43 INFO installing openblas-doc (0.3.29-r0)
2025/02/04 21:56:43 INFO installing ncurses-terminfo-base (6.5_p20241228-r0)
2025/02/04 21:56:43 INFO installing ncurses (6.5_p20241228-r0)
2025/02/04 21:56:43 INFO installing texinfo (7.2-r0)
2025/02/04 21:56:43 INFO installing wolfi-keys (1-r8)
2025/02/04 21:56:43 INFO installing busybox (1.37.0-r0)
2025/02/04 21:56:43 INFO installing wolfi-base (1-r6)
2025/02/04 21:56:44 INFO OCI layer digest: sha256:01b7ce102bcd6387cb04f535f79ee444337c4c6c7a6688377416c45233f8e6a0
2025/02/04 21:56:44 INFO OCI layer diffID: sha256:7aa1b3ff1a36a4cb8829eecd37603c16ae9860071619f28145c475f3b14d0ad9
2025/02/04 21:56:44 INFO saving OCI image locally: apko.local/cache:3239ce2df308a26ce3f748985bd3018755982de316d3d81bdf5dc6e6485f722e
2025/02/04 21:56:45 WARN skipping local domain tagging apko.local/cache:3239ce2df308a26ce3f748985bd3018755982de316d3d81bdf5dc6e6485f722e as index.docker.io/library/melange:latest
2025/02/04 21:56:45 INFO running step "test/docs"
2025/02/04 21:56:45 WARN + '[' -d /home/build ]
2025/02/04 21:56:45 WARN + cd /home/build
2025/02/04 21:56:45 WARN + exit 0
2025/02/04 21:56:45 INFO running step "docs readability check"
2025/02/04 21:56:45 WARN + '[' -d /home/build ]
2025/02/04 21:56:45 WARN + cd /home/build
2025/02/04 21:56:45 WARN + basename /home/build/melange-out/openblas-doc
2025/02/04 21:56:45 WARN + doc_pkg=openblas-doc
2025/02/04 21:56:45 WARN + apk info -L openblas-doc
2025/02/04 21:56:45 WARN + wc -l
2025/02/04 21:56:45 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 21:56:45 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 21:56:45 WARN + '[' 3 -le 3 ]
2025/02/04 21:56:45 WARN + echo See:
2025/02/04 21:56:45 WARN + apk info -L openblas-doc
2025/02/04 21:56:45 INFO See:
2025/02/04 21:56:45 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 21:56:45 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 21:56:45 INFO openblas-doc-0.3.29-r0 contains:
2025/02/04 21:56:45 INFO var/lib/db/sbom/openblas-doc-0.3.29-r0.spdx.json
2025/02/04 21:56:45 INFO
2025/02/04 21:56:45 WARN + echo 'This package [openblas-doc] is completely empty (i.e. installs no files).'
2025/02/04 21:56:45 WARN + echo 'Please check the package build for proper docs installation, and either:'
2025/02/04 21:56:45 WARN + echo '  (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or'
2025/02/04 21:56:45 WARN + echo '  (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or'
2025/02/04 21:56:45 WARN + echo '  (c) remove the docs subpackage entirely'
2025/02/04 21:56:45 WARN + exit 1
2025/02/04 21:56:45 INFO This package [openblas-doc] is completely empty (i.e. installs no files).
2025/02/04 21:56:45 INFO Please check the package build for proper docs installation, and either:
2025/02/04 21:56:45 INFO   (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or
2025/02/04 21:56:45 INFO   (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or
2025/02/04 21:56:45 INFO   (c) remove the docs subpackage entirely
2025/02/04 21:56:45 INFO pod a8a3f830b8ca80e1c57264c66c319d551381e67c734d951672ab067afdeb9370 terminated
2025/02/04 21:56:46 INFO pod 3624428d6e39d4b539d74e6424d73989ba36a6275dcb3848a2c27cd0a2c8300c terminated
2025/02/04 21:56:46 INFO ERROR: failed to test package. the test environment has been preserved:
2025/02/04 21:56:46 INFO   workspace dir: /home/user/tmp/melange-workspace-2856489356
2025/02/04 21:56:46 INFO   guest dir: /home/user/tmp/melange-guest-2129895668
2025/02/04 21:56:46 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:121: test/openblas] Error 1
```

--- a/openblas.yaml
+++ b/openblas.yaml
@@ -2,7 +2,7 @@
 package:
   name: openblas
   version: "0.3.29"
-  epoch: 0
+  epoch: 1
   description: fast BSD-licensed BLAS based on gotoBLAS2, with LAPACK
   copyright:
     - license: BSD-3-Clause
@@ -88,7 +88,13 @@ subpackages:
   - name: openblas-doc
     pipeline:
       - uses: split/manpages
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/share/doc/${{package.name}}
+          mv Changelog.txt TargetList.txt USAGE.md "${{targets.contextdir}}"/usr/share/doc/${{package.name}}/
     description: openblas manpages
+    test:
+      pipeline:
+        - uses: test/docs
 
   - name: liblapack
     pipeline:

--- a/smartypants.yaml
+++ b/smartypants.yaml
@@ -1,7 +1,7 @@
 package:
   name: smartypants
   version: 2.0.1
-  epoch: 3
+  epoch: 4
   description: Translate plain ASCII punctuation characters into “smart” typographic punctuation HTML entities
   copyright:
     - license: BSD-3-Clause
@@ -33,13 +33,22 @@ pipeline:
   - name: Python Install
     uses: python/install
 
+  # generate manpages
+  - runs: make -C docs man
+
   - uses: strip
 
 subpackages:
   - name: smartypants-doc
     pipeline:
       - uses: split/manpages
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/share/man/man1
+          mv docs/_build/man/smartypants.1 "${{targets.contextdir}}"/usr/share/man/man1/smartypants.1
     description: smartypants manpages
+    test:
+      pipeline:
+        - uses: test/docs
 
 update:
   enabled: true

